src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul {
+  margin: 0;
+}
+
+html {
+  font-size: 12px;
+  padding: 0 50px;
+  --active-color: #00acdc;
+}
+
+.header {
+  background-color: #ffff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  cursor: pointer;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
```

Comment:
better init font styles in html selector 
  font-family: "Roboto", sans-serif;
  font-style: normal;